src/AvPortalClient.php

```diff
@@ -76,6 +87,11 @@ public function query(array $options = []): ?array {
       'pagesize' => 15,
     ];
 
+    // Make sure that we are requesting a specified and supported asset type.
+    if (empty($options['type']) || !in_array($options['type'], self::$allowedTypes)) {
```

Comment:
If I ask "video,photo" as type, will code will force me to have reportage too. Let's:

- if the type option is empty, default it to the allowed types;
- if it's not empty, explode the list. If one non supported is found, throw an invalid argument exception saying that doesn't support that type.

Reply:
`If I ask "video,photo" as type, will code will force me to have reportage too.`
Sorry, it is my mistake. fixed.